worlds/musedash/Options.py

```diff
@@ -121,19 +122,16 @@ class MusicSheetWinCountPercentage(Range):
     display_name = "Music Sheets Needed to Win"
 
 
-class TrapTypes(Choice):
-    """This controls the types of traps that can be added to the pool.
+class ChosenTraps(OptionSet):
+    """This controls the types of traps that can be added to the pool. 
+    - Traps last the length of a song, or until you die.
     - VFX Traps consist of visual effects that play over the song. (i.e. Grayscale.)
     - SFX Traps consist of changing your sfx setting to one possibly more annoying sfx.
-    Traps last the length of a song, or until you die.
     Note: SFX traps are only available if [Just as Planned] DLC songs are enabled.
     """
-    display_name = "Available Trap Types"
-    option_None = 0
-    option_VFX = 1
-    option_SFX = 2
-    option_All = 3
-    default = 3
+    display_name = "Trap Types"
+    default = {}
+    valid_keys = [trap for trap in MuseDashCollections.trap_items.keys()]
```

Comment:
Unsure if this is intended, but the way these display on WebHost matches the order they are declared in `trap_items`, in other words, they aren't alphabetical.